src/server.js

```diff
@@ -202,6 +210,48 @@ app.get(
   }
 );
 
+app.get(
+  `${PREFIX_API}/:${URL_PARAM_EVENT_ID}/restaurants`,
+  getEvent,
+  async (request, response) => {
+    const { event } = request;
+    const users = event.users || {};
+    const userInput = Object.values(users);
```

Comment:
nit: rename userInput - this variable does not hold information that that user has input for this request, so let's give it a more descriptive name. this is part of what led to my misunderstanding in the other comment - I assumed you were getting the lat/lng from user input instead of the datastore because of the name of this variable.